### ISSUE TITLE
[REQUEST] Add a method to clear a RenderTexture

### ISSUE THREAD

**Issue by finscn** [pixijs/pixi.js#3595]
Now ,  The only way to clear a RenderTexture is  `renderer.render(foo, renderTexture, true)`.

But sometimes , I want to clear it without rendering anything.

So , I hope there could be a method to clear RenderTexture.

**Comment by finscn** [comment]
And why there is no:
```
renderTarget.renderTexture

or

renderTexture.renderTarget
```
?

**Comment by GoodBoyDigital** [comment]
currently you can do the following:
```
renderer.bindRenderTexture(myRenderTexture);
renderer.clear();
```

**Comment by finscn** [comment]
Could you consider to add `clear` to RenderTexture ?
And add some properties like this:
```
render.getCurrentRenderTexture();
render.getCurrentRenderTarget();

renderTarget.renderTexture
or
renderTexture.renderTarget

```

And it will override  current renderTexture.
There is no the api for getting current renderTexture, I can't do something like this:
```
var backup = renderer.getCurrentRenderTexture();

renderer.bindRenderTexture(myRenderTexture);
renderer.clear();

renderer.bindRenderTexture(backup);
```

**Comment by GoodBoyDigital** [comment]
I think a clear function sounds like a good idea!

```renderTexture.clear()```

Nice and easy :D

**Comment by finscn** [comment]
@GoodBoyDigital  , I found if no `renderer` , the RenderTexture can't clear itself.

So , I add the method to WebGLRenderer ,  then  `clearRenderTexture` & `bindRenderTexture` could be a couple.

**Comment by englercj** [comment]
Looks like this is fixed since #3647 was merged.
